components/automate-ui/src/app/components/chart-progress-bar/chart-progress-bar.component.ts

```diff
@@ -0,0 +1,123 @@
+import { Component, Input, OnInit, OnChanges } from '@angular/core';
+import { Styles } from './chart-progress-bar.model';
+
+@Component({
+  selector: 'app-chart-progress-bar',
+  templateUrl: './chart-progress-bar.component.html',
+  styleUrls: [ './chart-progress-bar.component.scss' ]
+})
+
+export class ChartProgressBarComponent implements OnInit, OnChanges {
+
+  @Input() size = 'large'; // large or small
+  @Input() value: number;
+  @Input() min: number;
```

Comment:
I can't see where the `this.min` is being used anywhere - do we still need it?

Reply:
@SEAjamieD as of now we're not using the min input value, so we can remove and add back if we decide to use min functionality. 👍🏻